src/tools/rustdoc-gui-test/src/main.rs

```diff
@@ -158,5 +161,5 @@ If you want to install the `browser-ui-test` dependency, run `npm install browse
 
     command.args(&config.test_args);
 
-    try_run(&mut command, config.verbose);
+    if try_run(&mut command, config.verbose) { Ok(()) } else { Err(()) }
```

Comment:
What about using `assert!` with a simple error message on all `try_run` usages?

Reply:
The error message would be less good because of the panic. I think `try_run` should return a `Result` instead of a boolean. Impossible to ignore it this way.